corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -187,6 +187,10 @@ def get_all_case_properties(app):
     return get_case_properties(app, app.get_case_types(), defaults=('name',))
 
 
+def get_all_case_properties_for_case_type(domain, case_type):
```

Comment:
How come you're using this function instead of just replacing `get_case_properties_for_case_type` with `all_case_properties_by_domain `?

Reply:
because `all_case_properties_by_domain` returns a dict, and the things that call this would want a list.